packages/manager/apps/dedicated/client/app/billing/orders/orders.routing.js

```diff
@@ -21,7 +21,7 @@ export default /* @ngInject */ ($stateProvider) => {
           .sort('date', 'DESC')
           .limit(5000)
           .execute(null, true)
-          .$promise.then(({ data }) => data),
+          .$promise.then(({ data }) => data.filter((row) => row != null)),
```

Comment:
Why does some results are null ? Isn't it an API issue to return non valid values ?

Reply:
Issue from iceberg, the api route itself doesn't return any null value. Waiting their response, I will convert this PR in draft